app/models/user.rb

```diff
@@ -12,4 +12,12 @@ class User < ActiveRecord::Base
   scope :therapists, -> { where(role_type: "Therapist") }
   scope :donors, -> { where(role_type: "Donor") }
   scope :admins, -> { where(role_type: "Administrator") }
+
+  def therapist?
```

Comment:
I think these predicates are defined already in [has_role.rb](https://github.com/ID8-neu/wediko-secret-snowman/blob/615d4cd1d0def6a8fa72ee63ab4e4eb9b8e513a5/app/models/concerns/has_role.rb)